src/stubs/win32/opamWindows.c

```diff
@@ -778,3 +779,28 @@ CAMLprim value OPAMW_GetConsoleWindowClass(value unit)
 
   CAMLreturn(result);
 }
+
+/* OPAMW_GetBinaryMode(channel) takes an out channel and returns its current
+ * mode, true if binary, false if text.
+ * It uses [caml_channel_binary_mode]. */
+CAMLprim value OPAMW_GetBinaryMode(value vchannel) {
+
+  CAMLparam1 (vchannel);
+  struct channel *channel = Channel(vchannel);
+
+  int i;
+
+#if OCAML_VERSION_MAJOR >= 5 && OCAML_VERSION_MINOR >= 2
```

Comment:
that comparison is incorrect, it will return false for ocaml 6.0 and 6.1 (I know...).
I believe that the correct way if `OCAML_VERSION >= 50200`

Reply:
```suggestion
#if OCAML_VERSION_MAJOR >= 50200
```

Reply:
well spotted, thanks!